pkg/dsl/task/process.go

```diff
@@ -0,0 +1,41 @@
+package task
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"cuelang.org/go/cue"
+)
+
+// Process processing the http task
+func Process(inst *cue.Instance) (*cue.Instance, error) {
+	taskVal := inst.Lookup("processing", "http")
+	resp, err := exec(taskVal)
+	if err != nil {
+		return nil, fmt.Errorf("fail to exec http task, %w", err)
+	}
+
+	appInst, err := inst.Fill(resp, "processing", "output")
+	if err != nil {
+		return nil, fmt.Errorf("fail to fill output from http, %w", err)
+	}
+	return appInst, nil
+}
+
+func exec(v cue.Value) (map[string]interface{}, error) {
+	htask := Lookup("http")
+	runner, err := htask(cue.Value{})
+	if err != nil {
+		return nil, err
+	}
+	got, err := runner.Run(&Meta{Obj: v})
+	if err != nil {
+		return nil, err
+	}
+	body := (got.(map[string]interface{}))["body"].(string)
```

Comment:
the assertion need to be checked, or it will panic the daemon Controller

Reply:
done